KomaMRIFiles/Project.toml

```diff
@@ -1,27 +1,25 @@
 name = "KomaMRIFiles"
 uuid = "fcf631a6-1c7e-4e88-9e64-b8888386d9dc"
 authors = ["Carlos Castillo Passi <cncastillo@uc.cl>"]
-version = "0.7.7"
+version = "0.8.0"
 
 [deps]
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
-JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
+KomaMRIBase = "d0bc0b20-b151-4d03-b2a4-6ca51751cb9c"
 MAT = "23992714-dd62-5051-b70f-ba57cb901cac"
 MRIFiles = "5a6f062f-bf45-497d-b654-ad17aae2a530"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
-KomaMRICore = "4baa4f4d-2ae9-40db-8331-a7d1080e3f4e"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 Scanf = "6ef1bc8b-493b-44e1-8d40-549aa65c4b41"
 
 [compat]
 FileIO = "1"
 HDF5 = "0.16, 0.17"
-JLD2 = "0.4"
+KomaMRIBase = "0.8"
 MAT = "0.10"
 MRIFiles = "0.1, 0.2"
```

Comment:
0.3?

Reply:
Check out the discussion on this at https://github.com/cncastillo/KomaMRI.jl/commit/a26b8bc7e297188e0e07ce65f2d63c1e72c14627#comments.

Reply:
We should be using the last version, "0.1, 0.2, 0.3"

Reply:
Tobias realeased the new version yesterday 😄!
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/f10b9faa8e594182cac8ca80ba9878576d86d07b